src/sentry/static/sentry/app/components/events/contexts.jsx

```diff
@@ -51,17 +51,21 @@ class ContextChunk extends React.Component {
     this.syncPlugin();
   }
 
-  componentDidUpdate(prevProps, prevState) {
+  componentDidUpdate(prevProps) {
     if (
-      prevProps.group.id !== this.props.group.id ||
-      prevProps.type !== this.props.type
+      prevProps.type !== this.props.type ||
+      (prevProps.group && this.props.group && prevProps.group.id !== this.props.group.id)
```

Comment:
@markstory so if `prevProps.group` is falsy, we don't ever want to call `syncPlugin`?

Reply:
Good catch, we want to sync if the new state is different than before. I'll get that fixed.

Reply:
@markstory In that case, you could use `_.get` here on `group.id` to simplify the code.